src/protocol-parser/protocol-parser.ts

```diff
@@ -62,10 +62,15 @@ export namespace Network {
   }
 
   export function parseContinueResponseParameters(params: unknown) {
+    // Work around of `cddlconv` https://github.com/google/cddlconv/issues/19.
```

Comment:
Let's not do it here but on `[BidiParser.ts](https://github.com/GoogleChromeLabs/chromium-bidi/pull/1691/files#diff-c1ecbf148f89ee8a59f541bbba9106c447b80f233024d396e906db77d06aa087)` as else it will get overridden by the bot.

Reply:
This file is not autogenerated. At least it does not have such a header.

Reply:
The auto-generated files are:
* `src/protocol-parser/generated/webdriver-bidi.ts`
* `src/protocol/generated/webdriver-bidi.ts`